processor/src/utils.ts

```diff
@@ -59,7 +59,7 @@ export function findKicadSchematic(inputDir, files, kitspaceYaml) {
 }
 
 export function isRelativeUrl(uri: string) {
-  return !uri.match(/^https?:\/\//)
+  return !uri.match(/^https?:\/\/|^mailto:/)
```

Comment:
I think it makes sense to make this generic over the protocol. We might get the same problem with e.g. `ssh://` links or something else.

Reply:
I found that the sanitizer only allows `http(s)` and `mailto`. I changed that :point_down:
https://github.com/kitspace/kitspace-v2/blob/f51a0792074fdbb19caaecccb1e5a68ebfb4907c/processor/src/tasks/processReadme/renderMarkdown.ts#L30

And changed the `isRelativeUrl` accordingly.

Do we need to allow any other protocols?

Reply:
No, we don't need them. It's fine and in fact better if the others don't get "linkified" at all.

Reply:
Actually, wait, maybe it handy to all of these linkified? I don't think they are used very much but might as well?

Reply:
They are already being linkified, no extra work is required. The only thing I did is stop treating them as relative URLs.

Reply:
But before the sanitizer would strip them out completely?

Reply:
Yes!